models/interface/account.model.interface.ts

```diff
@@ -7,6 +7,7 @@ export interface IAccount extends Document {
     password: string;
     companyId: Object;
     tenantId: string;
+    status: string;

```

Comment:
status type shouldn't be string, it should be of any enum type

Reply:
implemented